src/app/messages/new/page.tsx

```
@@ -0,0 +1,227 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/navigation";
+import { ArrowLeft, Search, User, MessageCircle } from "lucide-react";
+
+interface User {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)